docs/developerguide.rst

```diff
@@ -55,17 +54,17 @@ sources).
 
 .. code-block:: console
 
-   $ vim $HOME/reana/sources/reana-job-controller/reana_job_controller/app.py
+   $ vim $HOME/reana/sources/reana-server/reana_server/app.py
 
 4. Delete your pod and see a new pod being deployed with your changes:
 
 .. code-block:: console
 
-   $ kubectl get pods | grep job-controller
-   $ kubectl delete pod job-controller-2461563162-r4hgg
+   $ kubectl get pods | grep reana-server
```

Comment:
There is no pod named `reana-server` :wink: Please use `workflow-controller` instead.